torch/csrc/autograd/variable.h

```diff
@@ -109,6 +112,8 @@ struct VariableImpl : public at::TensorImpl {
   int output_nr;
   PyObject *pyobj;  // weak reference
 
+  std::string pretty_name;
```

Comment:
Can we just call it `name` everywhere in autograd for consistency between C++ and Python API?

Reply:
Well, PythonOp and CppOp inherit from Node and they both already have a name() method with a different meaning (the name of the underlying operation), so I preferred to not clash with that. 

If you don't mind that clash, or have some other suggestion, I'm happy to change it.

Reply:
I think `name_` should be safe, right? (and `name` in autograd, since there's no clash for that)